jargons/anonymous-function.md

```diff
@@ -0,0 +1,13 @@
+---
+title: anonymous function
+tags:
+- javascript
+---
+
+**İsimsiz fonksiyonlardır.** Fonksiyonlar [javascript](/k/javascript)'in temelleridir. Her bir fonksiyon bir görev yada bir değer hesaplayan ifade kümeleridir. Fakat bir fonksiyon bir isme sahip olmak zorunda değildir.
```

Comment:
"fonksiyonlar javascript'in temelleridir" önermesi oldukça iddialı olmuş. Bunu biraz daha yumuşatıp, "Fonksiyonlar, JavaScript içerisinde _first-class_ objelerdir; çünkü fonksiyonlar da diğer objeler gibi _property_ ve _method_'lar barındırabilirler." gibi bir şeye çevirebiliriz. Olduğu gibi yazmak zorunda değilsiniz (ki zaten yapmayın), kendinize göre düzenlerseniz harika olur. Önemli olan tek nokta, fonksiyonların _first-class object_ olduğu kısmı.

Reply:
evet bende şimdi tekrar okuyunca fazla iddalı gördüm :smile:  düzenliyorum